cupy/lib/_routines_poly.py

```diff
@@ -162,6 +161,7 @@ def _polypow_direct(x, n):
 
 
 def _polypow(x, n):
+    import cupyx.scipy.fft
```

Comment:
This can be moved to where `next_fast_len` is called

Reply:
Sure.